### DISCUSSION TITLE
Windows Arm32 support / V5.0

### DISCUSSION THREAD

**Discussion by ZeProgFactory** [TolikPylypchuk/SharpHook#54] [Q&A] [answered]
Hi,
1) If you drop Windows Arm32 support in SharpHook. Won’t we have an issue on programs running on Windows on a Mac with M1/M2. These processors are considered from the Windows side as ARM processors.
 
2) Do you have any timeframe for the release of version 5.0?
 
Continue the good work,
Thanks,
Michael

**Comment by TolikPylypchuk** [comment, marked as answer]
Hi! Thanks for asking!

1. There won't be problems with M1/M2 etc. since they use ARM64, not ARM32, and it's still supported.

2. I'm planning to release version 5.0.0 after I add tests and at least try to fix the last remaining open issue. I think I'll be done with that in the next couple weeks.
